frontend/src/components/Overview.tsx

```diff
@@ -78,7 +78,14 @@ export const OverviewContent: FC<OverviewContentProps> = ({
                   format={format}
                 />
               ) : (
-                <div className={classes(css.value)}>{value}</div>
+                <div
+                  className={classes(
+                    css.value,
+                    typeof format !== 'undefined' && css[format],
```

Comment:
when using the `classNames` library for classnames, you can pass an object for conditional classnames, i.e.
```
    className={classNames({
      [css.value]: true,
      [css.format]: format, // or !!format perhaps to convert to boolean if typescript wants it
    })}
```

Reply:
Also to check for undefined you can just put the value in the place of a boolean expression (like above) 
https://www.sitepoint.com/javascript-truthy-falsy/

unless I missed something and using `  typeof format !== 'undefined' && css[format]` was intentional

Reply:
You could even go a bit further with this and change the `[css.value]: true,` to
```
        className={classes(css.value, {
          [css.format]: format,
        })}
```

Reply:
This way you can separate the ones that get applied all the time from the conditional ones nicely